### DISCUSSION TITLE
Organizing External Modules inside the Oqtane.Server output directory

### DISCUSSION THREAD

**Discussion by vzdesic** [oqtane/oqtane.framework#1377] [Q&A] [answered]
I see that I can't organize external module's DLLs into subfolders inside Oqtane.Server output directory. I just checked. Modules are properly loaded only from root output directory.

Can we support it? Is it reasonable and possible to have it? What would be the effort of implementing such a feature? Any comment?

Thanks, Vedran

**Comment by sbwalker** [comment, marked as answer]
I investigated this previously and there are many recommendations by Microsoft to avoid using custom subfolders in the /bin. In general I would prefer to stick with more standard approaches to avoid obscure problems and future compatibility issues.
